src/components/loader/Loader.jsx

```diff
@@ -1,22 +1,14 @@
 import { ProgressBar } from "react-loader-spinner";
-import PropTypes from "prop-types";
-
-const Loader = ({ height, width, barColor, borderColor }) => {
-    Loader.propTypes = {
-        height: PropTypes.number.isRequired,
-        width: PropTypes.number.isRequired,
-        barColor: PropTypes.string.isRequired,
-        borderColor: PropTypes.string.isRequired,
-    };
 
+const Loader = () => {
```

Comment:
Should accept the same parameters, the only thing is that them should be initialized by default on the `Loader` component. Not only hardcode them directly.

Reply:
Done